src/core/Table/Table.tsx

```diff
@@ -589,6 +599,48 @@ export const Table = <
     [getPreparedRow, page],
   );
 
```

Comment:
Tests are needed.

Reply:
> Tests are needed.

```
const { container } = renderComponent({
    columns,
    enableDraggableColumns: true,
  });

  const headerCells = container.querySelectorAll<HTMLDivElement>(
    '.iui-table-header .iui-cell',
  );
  headerCells.forEach((cell) =>
    expect(cell.getAttribute('draggable')).toBe('true'),
  );

  const nameColumn = headerCells[0];
  const viewColumn = headerCells[2];

  fireEvent.dragStart(nameColumn);
  fireEvent.dragEnter(viewColumn);
  fireEvent.dragOver(viewColumn, { dataTransfer: { text: 0 } });
  fireEvent.drop(viewColumn);
```
I spent some time trying to make it work without any success.

Reply:
Can you push what you have so I could play around with the tests?

Reply:
Pushed fixed test. I had to mock `dataTransfer` object to make everything work.